commands/create.go

```diff
@@ -0,0 +1,101 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tarantool/cartridge-cli/common"
+	"github.com/tarantool/cartridge-cli/create"
+	"github.com/tarantool/cartridge-cli/project"
+	"github.com/tarantool/cartridge-cli/templates"
```

Comment:
Why templates dir is on top-level. I assume that we use templates only in "create". It should be placed inside "create" directory.